src/app/modules/design-patterns/observer/components/roulette-bet/roulette-bet.component.ts

```diff
@@ -0,0 +1,46 @@
+import { Component, Input, OnDestroy, OnInit } from '@angular/core';
+import { filter } from 'rxjs/operators';
+
+import { RoulettetBetModel } from '../../models/roulette-bet.model';
+import { RouletteService } from '../../services/roulette.service';
+
+
+@Component({
+  selector: 'app-roulette-bet',
+  templateUrl: './roulette-bet.component.html',
+  styleUrls: ['./roulette-bet.component.scss']
+})
+export class RouletteBetComponent implements OnInit, OnDestroy {
+  @Input() color: string;
+
+  betListArr: RoulettetBetModel[];
+
+  // TODO: Add type
+  rouletteSubscription$;
+
+  constructor(private readonly rouletteService: RouletteService) {}
+
+  ngOnDestroy(): void {
```

Comment:
Put this method after ngOnInit

Reply:
what about sorting alphabetically?

Reply:
dont do that, a good approach is sorting by lifeCycle order ;)